src/pages/account/mijn-bezoek-historiek/index.tsx

```diff
@@ -91,7 +91,7 @@ const AccountMyHistory: NextPage<DefaultSeoInfo> = ({ url }) => {
 			const response = await getAccessStatus(visit.spaceSlug);
 			switch (response?.status) {
 				case AccessStatus.ACCESS:
-					router.push(`/${visit.spaceSlug}`);
+					router.push(`/${ROUTE_PARTS.visit}/${visit.spaceSlug}`);
```

Comment:
shouldn't we redirect to this instead so we go directly to the correct url, instead of going through a redirect?
```suggestion
					router.push(`/${ROUTE_PARTS.search}?${VisitorSpaceFilterId.Maintainer}=${visitorSpace?.slug}`);
```